plone/outputfilters/browser/resolveuid.py

```diff
@@ -23,9 +23,7 @@ def uuidToURL(uuid):
         return res[0].getURL()
 
 
-@deprecate(
-    "Import from plone.app.uuid.utils instead. To be removed in version 6"
-)
+@deprecate("Import from plone.app.uuid.utils instead. To be removed in version 6")
```

Comment:
@jensens @mauritsvanrees per our deprecation policy (do we have such thing), as we did not remove it on time for Plone 6.0, can we delete it now, or should we bump the deprecation notice to 6.1 and try to remember next time? 🤔

Reply:
Damn. I would bump it to 6.1 to be sure.

Reply:
Ah, this is actually meant as: remove in version 6 of `plone.outputfilters`.  :-)  It is at version 5 now.

It might be more informative to mention a Plone version.  In `CMFPlone` lots of deprecation warnings for stuff moved to `plone.base` where we say: "Will be removed in Plone 7."
Maybe we should drop this one in Plone 7 as well?

Reply:
Ah, sure, confusing! So this is fine as is.

Reply:
Ok, will update it.

Would it make sense to add a check/quesiton on `zest.releaser` whenever you are about to make a major release (i.e. `6.0.0`) to ask to double check that all deprecations meant to be removed are actually removed? 🤔 

Otherwise we might be updating over and over these deprecation warnings 😓 🍀

Reply:
I update this one and the one above it that had no actual timeframe for when it would be removed.